app/test/features/chat/custom_input_test.dart

```diff
@@ -188,102 +187,6 @@ void main() {
         (ref, roomId) => Future.value(roomDrafts[roomId]),
       ),
     ];
```

Comment:
not good to remove unittest but it is only for the legacy chat, which we intend to remove anyways...

Reply:
I can revert it to comment as whole it as it was using `convienient_test_manager`.